### PR TITLE
Bibdk2021 1778 return user createdat

### PR DESCRIPTION
**Pull request by YousefMohsen**

### CODE REVIEW

File: src/__tests__/__snapshots__/schematransform.test.js.snap [hunk 1]
```diff
@@ -2715,8 +2715,13 @@ type User {
   name: String
   favoritePickUpBranch: String
 
+  \\"\\"\\"
+  Creation date in userdata service. Returns a timestamp with ISO 8601 format and in Coordinated Universal Time (UTC)
+  \\"\\"\\"
+  createdAt: DateTime
+
   \\"\\"\\"We can store userdata for more than 30 days if set to true.\\"\\"\\"
```
Comment (kiscdbc) [inline src/__tests__/__snapshots__/schematransform.test.js.snap:2723]: måske "we are allowed to"

**Review by jonaspraem — APPROVED** [review]

File: src/schema/user.js [hunk 2]
```diff
@@ -22,9 +22,13 @@ type User {
   name: String
   favoritePickUpBranch: String
   """
+  Creation date in userdata service. Returns a timestamp with ISO 8601 format and in Coordinated Universal Time (UTC)
+  """  
+  createdAt: DateTime
+  """
   We can store userdata for more than 30 days if set to true.
```
Comment (kiscdbc) [inline src/schema/user.js:29]: måske "we are allowed to"

File: src/schema/user.js [hunk 3]
```diff
@@ -215,6 +219,19 @@ function isCPRNumber(uniqueId) {
   return /^\d{10}$/.test(uniqueId);
 }
 
+/**
+ * Validates smaugUserId
+ * @param {String} uniqueId
```
Comment (kiscdbc) [inline src/schema/user.js:224]: string

**Review by kiscdbc — APPROVED** [review]

File: src/schema/user.js (hunk 2 above)
Comment (YousefMohsen) [reply src/schema/user.js]: fixed

File: src/schema/user.js (hunk 3 above)
Comment (YousefMohsen) [reply src/schema/user.js]: fixed

**Pull request merged**